cumulus_library/base_table_builder.py

```diff
@@ -69,6 +70,7 @@ def execute_queries(
                         )
 
                     table_name = table_name[0]
+                    # TODO: this may not be required? reinvestigate
```

Comment:
This PR is big enough that I'm deferring this to later - I'll create an issue for it